cli/src/semgrep/scan_report.py

```diff
@@ -3,9 +3,9 @@
 ##############################################################################
 # Helpers to run_scan.py to report scan status
 import sys
+from collections.abc import Sequence
```

Comment:
What is this change doing?

Reply:
I *think* one of the CI failures is related to this...

Reply:
huh. Not sure, I think this was some IDE autocompletion or similar. I'll fix.